rpms.in.yaml

```diff
@@ -8,6 +8,7 @@ contentOrigin:
 packages:
   - openssl
   - systemd
+#  - glibc-2.28-251.el8_10.16
 arches:
   - x86_64
   - aarch64
```

Comment:
I cannot comment on the lines below, but I think the key missing piece is to specify the target stage.
In your case it should be
```yaml
context:
  containerfile:
    file: Dockerfile.operator
    stageName: target-base
```

Reply:
+1